lib/zendesk_apps_support/package.rb

```diff
@@ -84,8 +85,12 @@ def files
       files
     end
 
+    def text_files
+      @text_files ||= files.select { |f| f =~ %r{.*((ht|x)ml?|js(on)?)$} } # match .html, .xml, .js and .json
+    end
+
     def js_files
-      @js_files ||= files.select { |f| f.to_s == 'app.js' || (f.to_s.start_with?('lib/') && f.to_s.end_with?('.js')) }
```

Comment:
I have a preference for the code as it was (i.e. not using a regex until it's the only option), but happy to go with this if others think it's nicer

Reply:
Yeah, fair. Just did this for consistency with the matchers surrounding it but not bothered either way